packages/pwa/app/pages/account/payments.test.js

```diff
@@ -84,42 +82,9 @@ const MockedComponent = () => {
     )
 }
 
-const server = setupServer(
-    rest.post('*/customers/actions/login', (req, res, ctx) =>
-        res(ctx.delay(0), ctx.set('authorization', `Bearer fakeToken`), ctx.json(mockCustomer))
-    ),
```

Comment:
Same, we're losing the `*/customers/actions/login` endpoint with the refactor, it looks we don't use it right now.
Would be a good idea to add it to the refactored `setupMockServer` in case we need it in future tests? 
Makes sense to me to remove it if we don't use tho.